pandas/io/parsers.py

```diff
@@ -1707,11 +1706,7 @@ def _convert_to_ndarrays(
                     or is_extension_array_dtype(cast_type)
                 ):
                     try:
-                        if (
```

Comment:
can the if go outside the try instead of the other way around?

Reply:
Done